femwell/waveguide.py

```diff
@@ -46,8 +46,16 @@ def mesh_waveguide(filename, wsim, hclad, hbox, wcore, hcore):
 
 if __name__ == '__main__':
     with tempfile.TemporaryDirectory() as tmpdirname:
-        mesh = mesh_waveguide(wsim=2, hclad=.7, hbox=.5, wcore=0.5, hcore=0.22, filename=tmpdirname + '/mesh.msh')
-        mesh = Mesh.load(tmpdirname + '/mesh.msh')
+        mesh = mesh_waveguide(
+            wsim=2,
+            hclad=0.7,
+            hbox=0.5,
+            wcore=0.5,
+            hcore=0.22,
+            filename=f'{tmpdirname}/mesh.msh',
+        )
+
```

Comment:
```suggestion
        mesh = mesh_waveguide(wsim=2, hclad=.7, hbox=.5, wcore=0.5, hcore=0.22, filename=f'{tmpdirname}/mesh.msh')
```
I'd guess breaking the parameters into lines would lower the readability, as the file focusses on implementing the simulation and the parameters for the example are not in focus here